akka-cluster/src/test/scala/akka/cluster/DowningProviderSpec.scala

```diff
@@ -76,17 +76,28 @@ class DowningProviderSpec extends AnyWordSpec with Matchers {
     }
 
     "stop the cluster if the downing provider throws exception in props method" in {
-      val system = ActorSystem(
-        "auto-downing",
-        ConfigFactory.parseString("""
+      // race condition where the downing provider failure can happen fast enough
+      // that creating the actor system throws on construcing thread,
```

Comment:
The exception will cause `postStop` in `ClusterDaemon` to run coordinated shutdown, which may run ActorSystem.shutdown before other things have been initialized.

An alternative solution for this test would be to configure:
```
akka.cluster.run-coordinated-shutdown-when-down = off
```